apps/wing/src/commands/compile.ts

```diff
@@ -196,7 +198,13 @@ export async function compile(entrypoint?: string, options?: CompileOptions): Pr
         output.push(causedBy.stack ?? "");
       }
 
-      throw new Error(output.join("\n"));
+      if (
+        (error as wingCompiler.PreflightError).causedBy.constructor.name === "NotImplementedError"
```

Comment:
Can we either hide these errors or show them as yellow one liners?

Reply:
<img width="1057" alt="image" src="https://github.com/winglang/wing/assets/39455181/89b0ddb6-47ce-4754-b2b9-c0af9743171e">
here is what the yellow looks like, what do you think?

Reply:
I wouldn't hide it, since saying something like "Tests: 5 unsupported" is giving too little information in my opinion (for what resource/method/props is unsupported for example)

Reply:
we can also make it white: 
<img width="1041" alt="image" src="https://github.com/winglang/wing/assets/39455181/1307404c-74d7-4a83-8a0c-5393947fa745">
<img width="1221" alt="image" src="https://github.com/winglang/wing/assets/39455181/4044cb93-1dae-47dd-97cc-f7ec99c54c9d">